index.html

```diff
@@ -7,15 +7,78 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
+    <!-- Add a link to your CSS file here (use the line above to guide you) -->su
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
+
 <body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<nav class = "navigation">
+  <img class = "logo" src="http://thinkful-ed.github.io/karma-clone/img/karma-logo.svg" alt = "Logo image" width = "26"/>
+  <ul class = "links">
+         <li><b>Meet Karma</b></li>
+         <li class ="style"> How it Works </li>
+         <li class ="style"> Store </li>
```

Comment:
the class name 'style' is not descriptive enough. Please use more descriptive names, like `nav-item`